# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/vlc/hotfix-mpc-decoder.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- a/modules/demux/mpc.c
+++ b/modules/demux/mpc.c
@@ -245,8 +245,6 @@ static void Close( vlc_object_t * p_this )
 #ifdef HAVE_MPC_MPCDEC_H
     if( p_sys->p_demux )
         mpc_demux_exit( p_sys->p_demux );
-#else
-    mpc_decoder_destroy( &p_sys->decoder );
 #endif
     free( p_sys );
 }
